trackData/projectManagement.json

```
@@ -51,20 +49,20 @@
       ],
       "examples": [
         "Oversaw technical delivery of Hightower",
-        "Managed infrastructure migration to VPC",
-        "Involved marketing, legal, and appropriate functions at project start"
+        "Managed data migration in production",
+        "Involved all appropriate functions at project start"
       ]
     }, {
       "points": 20,
-      "summary": "Manages major company pushes delivered by multiple teams",
+      "summary": "Manages major org pushes delivered by multiple teams",
       "signals": [
         "Considers external constraints and business objectives when planning",
         "Leads teams of teams, and coordinates effective cross-functional collaboration",
-        "Owns a key company metric"
+        "Owns a key team metric"
```

What does that mean? :thinking:

---

As far as I understand it, the followin are examples of key metrics:
- response-time and downtime of a microservice are two key metrics for that service team
- throughput and lead time (time between conception and delivery) of features are two key metric for a cross-functional/feature teams
- start render time is a key metric for a frontend team
- amount of donations is a key metric for fund-raising teams
and so on..

does that make sense? is it needed? can we rephrase it better?

---

Yes, but what does it mean to own the metric?

---

good question .. I think in reality those metrics are typically owned by teams. Sometimes certain individuals do appear to care/remember about them more than others, by setting them up, maintaining them (as in they are always being measured) and spend some time looking at them for the sake of making some analysis about the team performance.

Not sure if that's what owns would refer to here. I personally prefer teams to end up owning such metrics. It could start as one person pushing for and maintaining them, but it probably won't be that healthy nor sustainable to stay that way forever.

@RazShutyWMDE what do you think about this?

---

Thanks for the clarifications! Yes I tend to think it's a team responsibility. Same as making sure the sprint board represents reality. Yes, it might be that one person is thinking about it more than the others but it should be everyone's responsibility.

---

based on our discussion thus far, I'll try to rephrase it..

```suggestion
        "Pushes for and sets up a key metric for the team"
```